### PR TITLE
Add regression test for untitled scripts in Windows PowerShell

### PR DESCRIPTION
**Pull request by andschwa**
Specifically targeting the edge case where the last line of the script is a comment. Covers https://github.com/PowerShell/vscode-powershell/issues/3965

### CODE REVIEW

**Review by SeeminglyScience — APPROVED** [review]
LGTM! gj figuring this one out 😁 did not sound fun!

**Comment by andschwa** [comment]
@SeeminglyScience could you please re-review?

File: src/PowerShellEditorServices/Services/Workspace/WorkspaceService.cs [hunk 1]
```diff
@@ -152,8 +152,19 @@ public ScriptFile GetFile(DocumentUri documentUri)
...
+            try
+            {
+                return TryGetFile(new Uri(filePath), out scriptFile);
+            }
+            catch (UriFormatException)
+            {
+                scriptFile = null;
+                return false;
+            }
+        }
```
Comment (andschwa) [inline src/PowerShellEditorServices/Services/Workspace/WorkspaceService.cs:167]: I had to change this one to catch the exception instead of using `Uri.IsWellFormedUriString(path, UriKind.RelativeOrAbsolute)` since the path we were being given was like `\\foo\\bar`, which makes me hesitant about the above too.

File: test/PowerShellEditorServices.Test/Session/ScriptFileTests.cs [hunk 2]
```diff
@@ -651,6 +651,7 @@ public void DocumentUriReturnsCorrectStringForAbsolutePath()
         [InlineData("vscode-notebook-cell:/Users/me/Documents/test.ps1#0001", true)]
         [InlineData("https://microsoft.com", true)]
         [InlineData("Untitled:Untitled-1", true)]
+        [InlineData("'a log statement' > 'c:\\Users\\me\\Documents\\test.txt'\r\n", false)]
         public void IsUntitledFileIsCorrect(string path, bool expected) => Assert.Equal(expected, ScriptFile.IsUntitledPath(path));
```
Comment (andschwa) [inline test/PowerShellEditorServices.Test/Session/ScriptFileTests.cs:655]: See, this test implies that the function is asking "is this path a URI that indicates we're untitled?"

Comment (andschwa) [reply test/PowerShellEditorServices.Test/Session/ScriptFileTests.cs:655]: Perhaps we should expect this case to be `true` by flipping the logic with `IsWellFormedUri`. That is, if we're given something that is _not_ a URI, assume it's untitled, but I'm not sure that would work for the first two cases.

Comment (andschwa) [reply test/PowerShellEditorServices.Test/Session/ScriptFileTests.cs:655]: Yeah, doesn't work, since `/Users/me/Documents/test.ps1` is not a URI.
